grails-app/services/org/pih/warehouse/forecasting/ForecastingService.groovy

```diff
@@ -271,16 +270,39 @@ class ForecastingService {
                 DATE_FORMAT(date_issued, '%d/%b/%Y') as date_issued,
                 origin_name,
                 destination_name,
-                product_code,
+                product_demand_details.product_code,
```

Comment:
Why need for this only here?

Reply:
because there is field with the same name in the products table